emoji/src/main/java/com/vanniktech/emoji/EmojiButton.java

```diff
@@ -19,27 +18,12 @@ public EmojiButton(final Context context) {
 
   public EmojiButton(final Context context, final AttributeSet attrs) {
     super(context, attrs);
+    emojiSize = Utils.initTextView(this, attrs);
+  }
 
-    if (!isInEditMode()) {
-      EmojiManager.getInstance().verifyInstalled();
-    }
-
-    final Paint.FontMetrics fontMetrics = getPaint().getFontMetrics();
-    final float defaultEmojiSize = fontMetrics.descent - fontMetrics.ascent;
-
-    if (attrs == null) {
-      emojiSize = defaultEmojiSize;
-    } else {
-      final TypedArray a = getContext().obtainStyledAttributes(attrs, R.styleable.EmojiButton);
-
-      try {
-        emojiSize = a.getDimension(R.styleable.EmojiButton_emojiSize, defaultEmojiSize);
-      } finally {
-        a.recycle();
-      }
-    }
-
-    setText(getText());
```

Comment:
We don't need this anymore?

Reply:
We can use `emojiSize = Utils.initTextView(this, attrs);`